### PR TITLE
[15.0][MIG] stock_orderpoint_procure_location

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Supersedes https://github.com/OCA/stock-logistics-warehouse/pull/1464/

cc @ForgeFlow

### CODE REVIEW

File: stock_orderpoint_procure_location/__manifest__.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+    "version": "15.0.1.0.0",
     "author": "ForgeFlow, " "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/stock-logistics-warehouse",
     "category": "Warehouse Management",
-    "depends": ["purchase", "stock"],
+    "depends": ["purchase_stock"],
     "data": ["views/stock_warehouse_orderpoint_view.xml"],
     "license": "AGPL-3",
     "development_status": "Beta",
+    "maintainer": ["AaronHForgeFlow"],
```
Comment (rousseldenis) [inline stock_orderpoint_procure_location/__manifest__.py]: This is a list, so: `maintainers`

File: stock_orderpoint_procure_location/models/stock_warehouse_orderpoint.py [hunk 2]
```diff
@@ -13,11 +13,34 @@ class Orderpoint(models.Model):
...
+        res = super(Orderpoint, self)._prepare_procurement_values(date, group)
         if self.procure_location_id:
             res["procure_location_id"] = self.procure_location_id
         return res
+
+    def _quantity_in_progress(self):
+        res = super()._quantity_in_progress()
+        not_count_states = ["done", "cancel"]
+        for orderpoint in self:
+            if orderpoint.id == 780:
```
Comment (rousseldenis) [inline stock_orderpoint_procure_location/models/stock_warehouse_orderpoint.py]: ????

File: stock_orderpoint_procure_location/models/stock_warehouse_orderpoint.py [hunk 3]
```diff
@@ -13,11 +13,34 @@ class Orderpoint(models.Model):
...
+                        ("product_id", "=", orderpoint.product_id.id),
+                    ]
+                )
+                .mapped("move_ids")
+                .filtered(
+                    lambda m: m.location_dest_id == orderpoint.procure_location_id
+                    and m.state not in not_count_states
+                )
+            ):
+                res[orderpoint.id] += sm.product_uom._compute_quantity(
```
Comment (rousseldenis) [inline stock_orderpoint_procure_location/models/stock_warehouse_orderpoint.py:41]: Could you check why this is not covered by tests ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_orderpoint_procure_location/models/stock_warehouse_orderpoint.py (hunk 3 above)
Comment (AaronHForgeFlow) [reply stock_orderpoint_procure_location/models/stock_warehouse_orderpoint.py:41]: I will check that

**Comment by AaronHForgeFlow** [comment]
I will work on this, can someone remove the "stale" tag please?

**Pull request closed without merging**